ovrstat/player_stats.go

```diff
@@ -78,18 +76,16 @@ func playerStats(profilePath string, platform string) (*PlayerStats, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer res.Body.Close()
-	body, err := ioutil.ReadAll(res.Body)
-	if err != nil {
-		return nil, err
-	}
 
 	// Parses the stats request into a goquery document
-	pd, err := goquery.NewDocumentFromReader(bytes.NewReader(body))
+	pd, err := goquery.NewDocumentFromReader(res.Body)
```

Comment:
I like this, but if it fails to create a goquery document the reader will never be closed. We need to defer close before this.